### ISSUE TITLE
Cannot resolve symbol 'SwitcherItemLink'

### ISSUE THREAD

**Issue by MwSpaceLLC** [carbon-design-system/carbon-website#1300]
Hi there, think not important but i found this:

https://codesandbox.io/s/right-panel-with-switcher-ye8kg?file=/src/index.js:1547-1563

SwitcherItemLink component not found at 10.13v

![image](https://user-images.githubusercontent.com/29952045/84830083-263e8280-b029-11ea-96b7-228a5d96e510.png)

![image](https://user-images.githubusercontent.com/29952045/84830156-3eae9d00-b029-11ea-9a01-c8dde0db0108.png)

😅

**Comment by MwSpaceLLC** [comment]
Changed

SwitcherItem now imports Link and handles everything SwitcherItemLink did, mimicking HeaderMenuItem
Changed a few propTypes to handle the refactor

See => https://github.com/carbon-design-system/carbon/pull/3209
